CodeGeneration/Sources/SyntaxSupport/ExprNodes.swift

```diff
@@ -1683,37 +1683,50 @@ public let EXPR_NODES: [Node] = [
 
   // switch-expr -> identifier? ':'? 'switch' expr '{'
   //   switch-case-list '}' ';'?
-  //
-  // This node represents both a 'switch' expression, as well as a 'switch'
-  // statement when wrapped in an ExpressionStmt node.
   Node(
     kind: .switchExpr,
     base: .expr,
     nameForDiagnostics: "'switch' statement",
+    documentation: """
+      A `switch` statement expression.
+
+      This node represents both a 'switch' expression, as well as a 'switch'
+      statement when wrapped in an ``ExpressionStmtSyntax`` node.
+
+      ```swift
+      switch <#expression#> {
+      }
+      ```
```

Comment:
What is this snippet supposed to mean?

Reply:
The idea is to have a code-snip so the user can see an example if what the output is? 
Maybe it's just overhead. Not sure.

The idea comes from 
https://github.com/apple/swift-syntax/blob/9e798fa61ad386d9b59efe3d558fe1a734aa16c9/CodeGeneration/Sources/SyntaxSupport/DeclNodes.swift#L1232-L1239

Reply:
I'm was only a bit confused by the `<#expression#>` which immediately caught my attention. Thought it was the thing to look at principally. A more concrete example like in the initializer example would be better.

Reply:
I agree that a concrete example would be better here instead of the placeholder.